File: micro_services/micro_services/A00_Model/cResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace micro_services.A00
{
    public class cResponse
    {
        public string token { get; set; }
        public int message_code { get; set; }
        public string message { get; set; }
        public string data { get; set; }

    }
}

File: SimpleMRP/simplemrp/lib/models/cresponse.dart
class cResponse {
  String token="";
  int message_code=0;
  String message="";
  String data="";

  cResponse({this.token,this.message_code,this.message,this.data});

  factory cResponse.fromJson(Map<String, dynamic> parsedJson) {
    return cResponse(
        token: parsedJson['token'],
        message_code: parsedJson['message_code'],
        message: parsedJson['message'],
        data: parsedJson['data']);
  }

  Map<String, dynamic> toJson() {
    return {
      'token': token,
      'message_code': message_code,
      'message': message,
      'data': data,
    };
  }
}